readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page using grid
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_grid/)
+- [DEMO LINK](https://Stambuliak.github.io/layout_catalog_grid/)
```

Comment:
Do not add extra files (such as the `readme` file) to the PR

Reply:
deal